opentok/archives.py

```diff
@@ -157,11 +157,5 @@ def attrs(self):
     def json(self):
         return json.dumps(self.attrs(), default=dthandler, indent=4)
 
-    def __getitem__(self, key):
```

Comment:
"needed" is a strong word. it implements an API similar to the one you created in the initialization of the Callback object, where you can get each of the JSON properties as a property of the object. IMHO this is cleaner. and removing it is not an option without breaking the API. i don't see any strong reason to introduce a breaking change here.